styles.css

```diff
@@ -30,10 +25,49 @@ header {
 
 p {
   font-weight: 400;
+  font-size: 1.6rem;
+  line-height: 2.4rem;
   color: #344563;
   font-style: normal;
 }
 
+button {
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  padding: 1.2rem;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 1.7rem;
+  line-height: 2.4rem;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  background: none;
+}
+
+button:hover {
+  box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+  background: #6070ff;
+  color: #fff;
+}
+
+button:focus {
+  background: #2230d2;
+  color: #fff;
+}
+
+button:disabled,
+button[disabled] {
+  border: 1px solid #c1c7d0;
+  color: #5e6c84;
+}
```

Comment:
- [ ] [Optional]I would suggest you please remove `button:disabled` because it is affecting the the border and color of `Get my resume`

Reply:
@roshan-bajgain 

Thanks for reviewing and approving my work.

I added the 'button:disabled' to show the disabled feature.

I will remove it as suggested.

Thank you once again.